openrpc.json

```diff
@@ -1013,7 +1013,10 @@
         "title": "AddEthereumChainParameter",
         "type": "object",
         "required": [
-          "chainId"
+          "chainId",
+          "chainName",
+          "rpcUrls",
```

Comment:
`rpcUrls` should also in the schema [have `minItems: 1`](https://github.com/MetaMask/metamask-extension/blob/10996f0aaaf3a1d469cd79942c789e07152ad8f7/app/scripts/lib/rpc-method-middleware/handlers/add-ethereum-chain.js#L113), could also add `format: uri` for the url string item type

Reply:
done here: https://github.com/MetaMask/api-specs/pull/174/commits/f1d52358943714c58441d068f8089e82b2764247